tests/unittests/Foundation/NSThreadTests.m

```diff
@@ -57,15 +63,15 @@ - (void)countingloop {
     // THREAD_PRIORITY_HIGHEST and THREAD_PRIORITY_TIME_CRITICAL. Setting priority to half way between these makes it clear
     // what the priority is when conveyed back to the user.
 
-    EXPECT_EQ([myThread threadPriority], sExpectedHighThreadPriority);
+    EXPECT_LT([myThread threadPriority] - sExpectedHighThreadPriority, sMarginOfError);
```

Comment:
+1

---
In reply to: [73436715](https://github.com/Microsoft/WinObjC/pull/802#discussion_r73436715) [](ancestors = 73436715)